stubs/regex/regex/_regex.pyi

```diff
@@ -0,0 +1,192 @@
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> Match[AnyStr] | None: ...
+    def fullmatch(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> Match[AnyStr] | None: ...
+    def split(
+        self, string: AnyStr, maxsplit: int = ..., concurrent: bool | None = ..., timeout: float | None = ...
+    ) -> list[AnyStr | Any]: ...
+    def splititer(
+        self, string: AnyStr, maxsplit: int = ..., concurrent: bool | None = ..., timeout: float | None = ...
+    ) -> Splitter[AnyStr]: ...
+    def findall(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> list[Any]: ...
+    def finditer(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> Scanner[AnyStr]: ...
+    def sub(
+        self,
+        repl: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> AnyStr: ...
+    def subf(
+        self,
+        format: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> AnyStr: ...
+    def subn(
+        self,
+        repl: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> tuple[AnyStr, int]: ...
+    def subfn(
+        self,
+        format: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> tuple[AnyStr, int]: ...
+    def scanner(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: float | None = ...,
+    ) -> Scanner[AnyStr]: ...
+
+@final
+class Match(Generic[AnyStr]):
```

Comment:
Is this missing `__getitem__`? After updating some modules I now get hundreds of `"__getitem__" method not defined on type "Match[str]"` errors.

Reply:
I'm not using `regex` and have not investigated, but this seems likely. `re.Match` (or rather `typing.Match`) also has a `__getitem__()` field. Cc @jpy-git

Reply:
Makes sense, will make a PR 👍